@here/olp-sdk-dataservice-read/lib/client/index.ts

```diff
@@ -37,3 +37,4 @@ export * from "./QuadKeyPartitionsRequest";
 export * from "./SchemaRequest";
 export * from "./CatalogVersionRequest";
 export * from "./LayerVersionsRequest";
+export * from "./IndexLayerClient";
```

Comment:
Could you please change header to Copyright (C) 2019 - 2020 HERE Europe B.V.

Reply:
Done